storefront-reference-architecture-master/cartridges/app_storefront_assets_slots/cartridge/models/product/decorators/index.js

```diff
@@ -0,0 +1,8 @@
+var base = module.superModule;
+
+var originalExports = Object.keys(module.superModule);
+originalExports.forEach(function (originalExport) {
```

Comment:
Why do we need this? You can just get the super module, add your shipping and export it

Reply:
fixed